.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Then this will not change anything for you but those of us who do (Andre/Dave/Me) it will be very useful.